check_missing_installations.sh

```diff
@@ -40,6 +47,28 @@ exit_code=${PIPESTATUS[0]}
 
 ok_msg="Command 'eb --missing ...' succeeded, analysing output..."
 fail_msg="Command 'eb --missing ...' failed, check log '${eb_missing_out}'"
+if [ "$exit_code" -eq 1 ]; then
```

Comment:
Shouldn't this also be conditional on `[ "$#" -eq 2 ]`?

Reply:
Good point, we can't do that check if the second argument is missing. I'll fix it...

Reply:
Maybe just check for the existence of `pr_exceptions`, that's a bit more obvious

Reply:
I actually added `&& [ ! -z "$pr_exceptions" ]` instead. If no 2nd argument was passed, `pr_exceptions` will be empty and the `if` condition will be false. But _also_ if a 2nd argument was passed and there _were no `--from-pr` statements_ in the current PR, we can save ourselves the time and skip the second `eb --missing` check, as unmerged PRs cannot be the reason for failure in that scenario.